alerter.c

```diff
@@ -26,6 +26,9 @@ void alertInCelcius(float farenheit) {
 int main() {
     alertInCelcius(400.5);
     alertInCelcius(303.6);

```

Comment:
Atleast 3 test values need to be covered i.e. below threshold, threshold and above threshold. i.e. if threshold is 392 then consider values <392, =392 and >392, and correspondingly the behavior needs to be asserted